### ISSUE TITLE
Error fetching Image

### ISSUE THREAD

**Issue by tejas0145** [AdobeDocs/analytics.en#302]
Issue in ./help/components/segmentation/segmentation-workflow/seg-sequential-build.md

In Analytics documentation Under Include Everyone section one image is not loaded '70a875e2-0ef9-4459-8648-77c60081d64d'
showing status "Error Fetching Resource".

**Comment by gigazelle** [comment]
Are you still getting the error on this page? I believe that this error was due to temporary availability issues that happened on that specific day.

**Comment by tejas0145** [comment]
Yeah it's the same error. I think the image is not loaded by your side
that's why.

**Comment by tejas0145** [comment]
<img width="765" alt="Screenshot 2025-01-09 at 10 05 57" src="https://github.com/user-attachments/assets/4fcbf6ea-1e76-4e74-9667-cb8dbe0e4ad3" />
Here is that page screenshot which image is been not loaded by your side. From my side I had cleared the cache as well as done hard reload but the result is in that image.

**Comment by ridmaur** [comment]
We have updated this page and the issue with the image is solved.
